src/main/scala/lantern/dslapi.scala

```diff
@@ -571,16 +571,13 @@ void Snippet(char *);
 std::random_device rd{};
```

Comment:
`rd`, `gen`, and `d` can be moved into `Backend.setup()` now.
I'll hold off on this though.